### ISSUE TITLE
[FP] 'str(i)str(...)' could be replaced with 'str(i)pos(...)': processes true when expected to process false only

### ISSUE THREAD

**Issue by dphadtare** [kalessil/phpinspectionsea#820]
**What steps will reproduce the problem?**
```
if( true == stristr( 'digital_signature', 'digital' ) ) { 
    echo 'Yes'; 
}
```
1. Add this code in your file
2. Run inspection
3. Apply suggested quick fix
4. What is the expected result?
```
if( false !== stripos( 'digital_signature', 'digital' ) ) { 
    echo 'Yes'; 
}
```
**What happens instead?**
line will be changed to 
```
if( false === stripos( 'digital_signature', 'digital' ) ) { 
    echo 'Yes'; 
}
```
**code sample**
```
<?php 
if( true == stristr( 'digital_signature', 'dig' ) ) { 
    echo 'Yes here'; 
} 

echo '=='; 

if( false != stristr( 'digital_signature', 'digital' ) ) { 
    echo 'Yes here1'; 
} 
echo '==='; 

if( false === stripos( 'digital_signature', 'dig' ) ) { 
    echo 'Yes here'; 
}
?>
```

**Comment by kalessil** [comment]
Thank you for reporting @dphadtare. That's a false-positive and it has been fixed - coming with the next release.
